storey/sources.py

```diff
@@ -1012,3 +1012,99 @@ def _init(self):
             else:
                 df = pandas.read_parquet(path, columns=self._columns, storage_options=self._storage_options)
             self._dfs.append(df)
+
+
+class SQLSource(_IterableSource, WithUUID):
+    """Use SQL table as input source for a flow.
+
+
+    :parameter key_field: the primary key of the table.
+    :parameter time_field: column to be used as time for events.
+    :parameter id_field: column to be used as ID for events.
+    :parameter db_name: url string connection to sql database.
+    :parameter table_name: the name of the table to access,
+                                from the current database
+
+    """
+
+    def __init__(
+        self,
+        db_path: str = None,
+        table_name: str = None,
+        key_field: Optional[Union[str, List[str]]] = None,
```

Comment:
If you want the default to be None, then the type annotation should be
```suggestion
        key_field: Optional[Union[None, str, List[str]]] = None,
```
(or `NoneType` if you want to be pedantic)